src/main/java/Duke.java

```diff
@@ -1,10 +1,93 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
+        int count = 0;
+        String input;
+        Task list[] = new Task[100];
```

Comment:
Perhaps can name the Task array to "tasks" instead of "list", this name probably will be easier to understand by other developers